source/common/router/upstream_request.cc

```diff
@@ -388,6 +388,18 @@ void UpstreamRequest::acceptHeadersFromRouter(bool end_stream) {
         upstreamLog(AccessLog::AccessLogType::UpstreamPeriodic);
         resetUpstreamLogFlushTimer();
       }
+      // Both downstream and upstream bytes meters may not be initialized when
+      // the timer goes off, e.g. if it takes longer than the interval for a
```

Comment:
why dont we need to account bytes in acceptDataFromRouter as well?

Reply:
They are accounted for. This section is in the context of the timer callback, which will be run periodically.